brand-guidelines/color-principles.md

```diff
@@ -140,3 +140,16 @@ Tints expand upon the primary and secondary color palettes to help create visual
     {% endfor %}
     </table>
 {:/nomarkdown}
+
+
+## Updating the color palette
+
+Follow these instructions to update the color palette within the Design Manual
```

Comment:
TRIVIAL: instructions appears implied by the list, so this could be `To update the color palette within the Design Manual:` (also trailing colon added.)

Reply:
Amended my commit with this suggestion.